port/unix/omrmmap.c

```diff
@@ -88,6 +176,12 @@ omrmmap_map_file(struct OMRPortLibrary *portLibrary, intptr_t file, uint64_t off
 	J9MmapHandle *returnVal;
 	OMRMemCategory *category = omrmem_get_category(portLibrary, categoryCode);
 
+#if defined(J9ZOS390)
+	if (OMR_ARE_ALL_BITS_SET(flags, OMRPORT_MMAP_FLAG_ZOS_READ_MAPFILE)) {
```

Comment:
This (and the test on line 214) should probably use `OMR_ARE_ANY_BITS_SET()` instead.